stable_baselines/common/vec_env/dummy_vec_env.py

```diff
@@ -68,37 +69,19 @@ def _save_obs(self, env_idx, obs):
     def _obs_from_buf(self):
         return dict_to_obs(self.observation_space, copy_obs_dict(self.buf_obs))
 
-    def env_method(self, method_name, *method_args, **method_kwargs):
-        """
-        Provides an interface to call arbitrary class methods of vectorized environments
-
-        :param method_name: (str) The name of the env class method to invoke
-        :param method_args: (tuple) Any positional arguments to provide in the call
-        :param method_kwargs: (dict) Any keyword arguments to provide in the call
-        :return: (list) List of items retured by the environment's method call
-        """
-        return [getattr(env_i, method_name)(*method_args, **method_kwargs) for env_i in self.envs]
+    def env_method(self, method_name, indices=None, *method_args, **method_kwargs):
```

Comment:
Needs documentation. I'd be OK with a one-sentence description pointing to the VecEnv base class (although @erniejunior may be more stringent), but it shouldn't be empty.